src/concurrency.md

```diff
@@ -223,8 +256,11 @@ Lastly, collects calculation results via [`mpsc::channel`] with [`Receiver::recv
 [`Receiver::recv`]: https://doc.rust-lang.org/std/sync/mpsc/struct.Receiver.html#method.recv
 [`Rgb::from_channels`]: https://docs.rs/image/*/image/struct.Rgb.html#method.from_channels
 [`Scope::spawn`]: https://docs.rs/crossbeam/0.*/crossbeam/struct.Scope.html#method.spawn
+[`stable sorting`]: https://docs.rs/rayon/0.8/rayon/slice/trait.ParallelSliceMut.html#method.par_sort
```

Comment:
I am worried that the API might change.
This particular function was added in the latest `0.8.2`.
If you think that it is ok, I can switch it to the wildcard.